vault/identity_store_aliases.go

```diff
@@ -138,6 +138,7 @@ func (i *IdentityStore) handleAliasUpdateCommon() framework.OperationFunc {
 			}
 		} else {
 			alias.Name = aliasName
```

Comment:
Same, should we sanitize this here?

Reply:
Same here. We don't need to sanitize here.